app/src/androidTest/java/com/github/se/cyrcle/ui/report/ImageReportScreen.kt

```diff
@@ -0,0 +1,130 @@
+package com.github.se.cyrcle.ui.report
+
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.se.cyrcle.di.mocks.*
+import com.github.se.cyrcle.model.parking.ParkingViewModel
+import com.github.se.cyrcle.model.user.TestInstancesUser
+import com.github.se.cyrcle.model.user.UserViewModel
+import com.github.se.cyrcle.ui.navigation.NavigationActions
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito
+
+@RunWith(AndroidJUnit4::class)
+class ImageReportScreenTest {
```

Comment:
Be careful here the title of the file can be confusing because you forgot the Test word

Reply:
Oops, I'll fix!